packages/stk/stk_middle_mesh/cmake/Dependencies.cmake

```diff
@@ -1,8 +1,8 @@
-SET(LIB_REQUIRED_DEP_PACKAGES STKSearch)
+SET(LIB_REQUIRED_DEP_PACKAGES STKUtil STKSearch)
 SET(LIB_OPTIONAL_DEP_PACKAGES)
 SET(TEST_REQUIRED_DEP_PACKAGES Gtest)
-SET(TEST_OPTIONAL_DEP_PACKAGES)
-SET(LIB_REQUIRED_DEP_TPLS)
+SET(TEST_OPTIONAL_DEP_PACKAGES STKMesh STKIO)
+SET(LIB_REQUIRED_DEP_TPLS CDT)
```

Comment:
Here is the problem.  If `CDT` really is a required dependency then `STKMiddle_Mesh` needs to be `EX	` as well.